debug/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "debug",
+  "private": true,
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "run-with-download": "pnpm dlx @workleap/create-schemas arg1 arg2",
-    "run-from-dev-dep": "pnpm create-schemas args1 args2"
+    "run": "create-schemas args1 args2"
   },
   "devDependencies": {
-    "@workleap/create-schemas": "0.0.1-preview.0"
+    "@workleap/create-schemas": "workspace:*"
```

Comment:
No need to do package linking manually. We can just use a [pnpm workspace](https://pnpm.io/workspaces).